htdocs/searchidle.php

```diff
@@ -15,7 +15,8 @@
 $dateIdleLdap = string2ldapDate( $dateIdle->format('d/m/Y') );
 
 # Search filter
-$ldap_filter = "(&".$ldap_user_filter."(|(!(authTimestamp=*))(authTimestamp<=".$dateIdleLdap.")))";
+$lastauth_attr = $ldap_lastauth_attribute ? $ldap_lastauth_attribute : "authTimestamp";
```

Comment:
I don't like deprecated values in the code (authTimestamp), but I don't have any other way to deal with the update.

Reply:
Indeed, the other solution is to set the value in config.inc.php, as we recommend to use a config.inc.local.php to override it. And we can mention this in upgrade notes.